src/hello/src/lib.rs

```diff
@@ -1,12 +1,14 @@
-#![feature(proc_macro, wasm_custom_section, wasm_import_module)]
+#![feature(proc_macro, wasm_custom_section, wasm_import_module, proc_macro_path_invoc)]
```

Comment:
I think `proc_macro_path_invoc` is unnecessary since the json bits have been removed.